pysatNASA/instruments/icon_mighti.py

```diff
@@ -13,8 +13,8 @@
 tag
     Supports 'los_wind_green', 'los_wind_red', 'vector_wind_green',
     'vector_wind_red', 'temperature'.  Note that not every data product
-    available for every sat_id
-sat_id
+    available for every inst_id
+inst_id
     '', 'a', or 'b'
```

Comment:
Recommend changing '' inst_id to 'vect'.

Reply:
updated as 'vector'